app/views/govuk_publishing_components/components/_intervention.html.erb

```diff
@@ -0,0 +1,24 @@
+<%
+  local_assigns[:margin_bottom] ||= 8
+  local_assigns[:margin_bottom] = 8 if local_assigns[:margin_bottom] > 9
+
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+
+  css_classes = %w[gem-c-intervention]
+  css_classes << (shared_helper.get_margin_bottom)
+
+  aria_attributes = {label: 'Intervention'}
```

Comment:
I don't think that the name "Intervention" is that useful - in VoiceOver it reads out "Intervention, Region".

I'd suggest removing the `aria-label` attribute completely; or using `aria-labelledby` to point to the heading inside the region; or replacing the content of `aria-label` with something that better explains the region such as "Suggested pages".

Reply:
Good point. I'll remove the aria-label for now since the explanation is in the text already.